NTupleMaker/test/mutau_TemplateDir/analysisMacroSynch_lept_mt_MC17.conf

```diff
@@ -78,9 +77,8 @@ dzMuonCut = 0.2
 ApplyMuonId = true
 
 # tau cuts
-ptTauLowCut = 20.0
-ptTauHighCut = 20.0
-etaTauCut = 2.1
+ptTauLowCut = 30.0
+etaTauCut = 2.3
 dzTauCut = 0.2
```

Comment:
Until we fix the scale factors for the triggers the range cannot reach 2.3, as in those files the range was erroneously set to 2.1. By extending the range to 2.3 we add events for which the trigger weight is not defined.

Reply:
thanks for pointing that out. But then I don't understand why they quote this number in the analysis note, and quote it as well on the twiki:
https://twiki.cern.ch/twiki/bin/view/CMS/HiggsToTauTauWorkingLegacyRun2

Also, please keep in mind that the data/MC agreement seemed to improve when all these changes were implemented..
Hope all makes sense ;-)

Reply:
If I understand Andrea correctly, you should set it to 2.1 for now since the trigger weights are not correct otherwise. In principal 2.3 would be the correct eta range to use, but I would update this only after the corrected scale factors arrived.